index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Cats, Cats, and More Cats</title>
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./responsive.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <main>
+      <article class="post">
+        <header class="post__header">
+          <div class="post__header__logo">Another Cat Site</div>
```

Comment:
While this is a fine style, I recommend going this route
```css
post__headerLogo
```
or, if you don't like the BEM syntax,
```css
post-header-logo
```
Regardless, as long as you are consistent, then :+1: